use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -124,35 +143,28 @@ EOF
 $ sudo clr-boot-manager update
 ```
 
-### Replace Clear Linux kernel with LTS kernel
+### Replace Clear Linux kernel with LTS kernel (Optional)
+
+If you plan to replace the host driver then the LTS kernel is necessary. This
+installs the Linux LTS kernel version `4.19` from 2018 because the 
```

Comment:
> kernel ... from 2018

This sounds somewhat alarming given the pace of change in the kernel. Maybe the wording could be clarified to state this is the "latest 4.19 LTS kernel"?

Reply:
The reason is Clear Linux specific. The Clear Linux 5.x kernel has some of the kernel options set such that you can't load unsigned kernel modules that QAT requires. The LTS kernel doesn't have this problem. I was not sure how much I should explain that. I am happy to say latest LTS kernel.

Reply:
Perhaps check whether you can use 4.19 LTS upstream host drivers (similar to linux 5.x)? If yes, we could skip talking about host drivers at all.

Reply:
I decided to remove this section since  I only kept it for the instructions on how to compile and load out of tree drivers. You can't do this with the 5.x clear linux kernel due to how it is configured.